packages/manager/cypress/e2e/core/linodes/migrate-linode.spec.ts

```diff
@@ -0,0 +1,253 @@
+    mockGetLinodeType(dcPricingMockLinodeTypes[0]).as('getLinodeTypeOld');
+    mockGetLinodeType(dcPricingMockLinodeTypes[1]).as('getLInodeTypeNew');
+
+    cy.visitWithLogin(`/linodes/${mockLinode.id}?migrate=true`);
+    cy.wait([
+      '@getClientStream',
+      '@getFeatureFlags',
+      '@getLinode',
+      '@getLinodeDisks',
+      '@getLinodeVolumes',
+    ]);
+
+    ui.button.findByTitle('Enter Migration Queue').should('be.disabled');
+    cy.findByText(`${initialRegion.label}`).should('be.visible');
+    getClick('[data-qa-checked="false"]');
+    cy.findByText(`North America: ${initialRegion.label}`).should('be.visible');
+    containsClick(selectRegionString);
+
+    ui.regionSelect.findItemByRegionLabel(newRegion.label).click();
+
+    cy.findByText(dcPricingCurrentPriceLabel).should('not.exist');
+    const currentPrice = dcPricingMockLinodeTypes[0].region_prices.find(
+      (regionPrice) => regionPrice.id === initialRegion.id
+    );
+    cy.get('[data-testid="current-price-panel"]').should('not.exist');
+    cy.findByText(`$${currentPrice.monthly.toFixed(2)}`).should('not.exist');
+    cy.findByText(`$${currentPrice.hourly}`).should('not.exist');
+    cy.findByText(
+      `$${dcPricingMockLinodeTypes[0].addons.backups.price.monthly.toFixed(2)}`
+    ).should('not.exist');
+
+    cy.findByText(dcPricingNewPriceLabel).should('not.exist');
+    const newPrice = dcPricingMockLinodeTypes[1].region_prices.find(
+      (linodeType) => linodeType.id === newRegion.id
+    );
+    cy.get('[data-testid="new-price-panel"]').should('not.exist');
+    cy.findByText(`$${newPrice.monthly.toFixed(2)}`).should('not.exist');
+    cy.findByText(`$${newPrice.hourly}`).should('not.exist');
+    cy.findByText(
+      `$${dcPricingMockLinodeTypes[1].addons.backups.price.monthly.toFixed(2)}`
+    ).should('not.exist');
+
+    // intercept migration request and stub it, respond with 200
+    cy.intercept(
+      'POST',
+      apiMatcher(`linode/instances/${mockLinode.id}/migrate`),
+      {
+        statusCode: 200,
+      }
+    ).as('migrateReq');
+
+    fbtClick('Enter Migration Queue');
+    cy.wait('@migrateReq').its('response.statusCode').should('eq', 200);
+  });
+
+  /*
+   * - Confirms DC-specific pricing UI flow works as expected during Linode migration.
+   * - Confirms that pricing comparison is shown in "Region" section when migration occurs in different DCs.
+   */
+  it('shows DC-specific pricing information when migrating linodes between different DCs', () => {
+    const initialRegion = getRegionById('us-west');
+    const newRegion = getRegionById('us-east');
+
+    const mockLinode = linodeFactory.build({
+      region: initialRegion.id,
+      type: dcPricingMockLinodeTypes[0].id,
+    });
+
+    const mockLinodeDisks = linodeDiskFactory.buildList(3);
+
+    mockGetLinodeDetails(mockLinode.id, mockLinode).as('getLinode');
+
+    // TODO Remove feature flag mocks once DC pricing is live.
```

Comment:
```suggestion
    // TODO: DC Pricing - M3-7073: Remove feature flag mocks once DC pricing is live.
```

For clarity and ease of clean up later, let's format DC pricing related TODOs with `TODO: DC Pricing`. In this case, we have a related ticket number, so it's useful to include that too.